test/mpi/.gitignore

```diff
@@ -1096,12 +1065,9 @@
 /io/resized
 /io/setinfo
 /io/setviewcur
-/io/test.ord
-/io/testfile*
```

Comment:
these are valid. they are temporary files created by the I/O interface tests.

Reply:
I see `io/testfile.*` is moved to the new block and then moved back in the sorting commit. but `io/test.ord` should not be removed.